src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -285,11 +286,29 @@ def create_event_unique_user_frequency_data_condition(
 def create_percent_sessions_data_condition(
     data: dict[str, Any], dcg: DataConditionGroup
 ) -> DataConditionKwargs:
-    return create_base_event_frequency_data_condition(
-        data=data,
-        dcg=dcg,
-        count_type=Condition.PERCENT_SESSIONS_COUNT,
-        percent_type=Condition.PERCENT_SESSIONS_PERCENT,
+    comparison_type = data.get(
+        "comparisonType", ComparisonType.COUNT
+    )  # this is camelCase, age comparison is snake_case
+    comparison_type = ComparisonType(comparison_type)
+
+    value = float(data["value"])
```

Comment:
i need this as float

Reply:
🤔 could we DRY this up by having `create_base_event_frequency_data_condition` and `create_base_event_frequency_data_condition` be something like:
```py
def create_base_event_frequency_data_condition(...):
    value = float(data["value"])
    create_frequency_event_data_condition(....)
```
and
```py
def create_base_event_frequency_data_condition(...):
    value = int(data["value"])
    create_frequency_event_data_condition(....)
```

then define `create_frequency_event_data_condition` in a way that it doesn't cast the data value?